lib/audio/music_service.dart

```diff
@@ -0,0 +1,98 @@
+import 'dart:async';
+
+import 'package:audioplayers/audioplayers.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_game_challenge/settings/settings.dart';
+
+class MusicService {
+  final AudioPlayer _musicPlayer = AudioPlayer()..setVolume(0.4);
```

Comment:
Could you please provide AudioPlayer through the constructor properties. Will be handful when we will wright a test for the MusicService

Reply:
done